src/styles/components/expanding-table/styles.less

```diff
@@ -85,6 +85,10 @@
 
   .expanding-table-child {
 
+    > div {
```

Comment:
I see. Yeah, `@expanding-table-line-height` is a bit bigger...I was just curious where the 25px came from or if there was a better constant to use.